### PR TITLE
Add files upload to rooms

### PR DESCRIPTION
**Pull request by MrRTi**
This PR adds ability to upload files using [rooms.upload](https://developer.rocket.chat/reference/api/rest-api/endpoints/rooms-endpoints/upload-file-to-a-room) endpoint

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -80,3 +80,6 @@ Style/RedundantRegexpEscape:
 
 Style/SlicingWithRange:
   Enabled: true
+
+Gemspec/DevelopmentDependencies:
+  EnforcedStyle: gemspec
```
Comment (abrom) [inline .rubocop.yml:85]: This has been addressed in `main`. Can you please rebase and revert this change out

**Review by abrom — CHANGES_REQUESTED** [review]
Change looks good @MrRTi 👍 

One minor thing, otherwise good to go

File: .rubocop.yml (hunk 1 above)
Comment (MrRTi) [reply .rubocop.yml]: I've done rebase and removed changes in .rubocop.yml

**Review by abrom — APPROVED** [review]
Looks great, thanks @MrRTi 👍

**Comment by abrom** [comment]
Released in [v0.2.3](https://rubygems.org/gems/rocketchat/versions/0.2.3)
